### DISCUSSION TITLE
can't connect lcd2004 to blue pill

### DISCUSSION THREAD

**Discussion by retroretro954** [davetcc/tcMenu#422] [Q&A]
I've been strugling for a week already with conecting lcd2004 with pcf backpack to stm32f103c8. Could you please help me. I tried a lot of options but nothing works. Can't understand what's wrong

**Comment by retroretro954** [comment]
I finally managed to connect it, but anyway, thank you!

****

**Reply by davetcc** [reply]
We test every release on STM32Duino so there should be no issue there.

But in order to get a response, you need to provide us with enough details of what you've done so we can help. IE in this case you're trying to get I2C working:

* Output of the I2C scanner sketch on your board (IE where it found the LCD).
* The pins you're trying to connect on, and some kind of documentation you read that suggested those pins were available as I2C.
* At least a snippet from the sketch showing that you've correctly configured I2C.
